app/models/article.rb

```diff
@@ -1,4 +1,8 @@
 class Article < ApplicationRecord
+  self.ignored_columns = %w[
+    support_webmentions
```

Comment:
I'm assuming the plan is to remove this so we don't have to keep it in sync and instead check the canonical URL on the fly, is that correct?